llvm/test/tools/llvm-gsymutil/X86/elf-empty-dir.yaml

```diff
@@ -0,0 +1,108 @@
+## Test converting DWARF using relative path
+
+# RUN: yaml2obj %s -o %t
+# RUN: llvm-gsymutil --convert %t -o %t.gsym --segment-size=100000 --num-threads=80 --quiet 2>&1 | FileCheck %s --check-prefix=CONVERT
```

Comment:
Don't specify the number of threads, they will automatically be set to a good value.